configure.ac

```diff
@@ -102,10 +102,16 @@ PKG_CHECK_MODULES([JSONC], [json-c],,
 AC_SUBST(JSONC_CFLAGS)
 AC_SUBST(JSONC_LIBS)
 
-PKG_CHECK_MODULES([TIRPC], [libtirpc],,
-                  [AC_MSG_ERROR([tirpc library is required to build gluster-block])])
-AC_SUBST(TIRPC_CFLAGS)
-AC_SUBST(TIRPC_LIBS)
+if test "$enable_tirpc" != "no"; then
+    enable_tirpc="yes";
```

Comment:
We may need to add the help promotion, something like:

$ ./configure --help
[...]
--enable-tirpc Please enable litirpc library if glibc >= 2.26 [default=yes]
[...]

Will this make sense ?

Reply:
Hmm, Okay! We can add a help string.

Will refresh this in the next spin.

Thanks!

Reply:
@lxbsz please check:
✨ ./configure --help | grep tirpc
  --enable-libtirpc       enable use of tirpc [default: yes]

Reply:
Yeah, cool. it works as expected.